forch/forchestrator.py

```diff
@@ -45,6 +45,7 @@
 _FAUCET_PROM_PORT_DEFAULT = 9302
 _GAUGE_PROM_HOST = '127.0.0.1'
 _GAUGE_PROM_PORT_DEFAULT = 9303
+_CONFIG_HASH_MAX_RETRY = 5
```

Comment:
make configurable (externally) with default to 5

Reply:
Added _DEFAULT